.stylelintrc.json

```diff
@@ -1,11 +1,17 @@
 {
-    "extends": [
-        "stylelint-config-sass-guidelines"
-    ],
-    "rules": {
-        "max-nesting-depth": 4,
-        "selector-no-qualifying-type": null,
-        "selector-max-compound-selectors": 4,
-        "order/properties-alphabetical-order": null
-    }
+  "plugins": [
+    "stylelint-no-unsupported-browser-features"
+  ],
+  "extends": [
+    "stylelint-config-sass-guidelines"
+  ],
+  "rules": {
+    "max-nesting-depth": 4,
+    "selector-no-qualifying-type": null,
+    "selector-max-compound-selectors": 4,
+    "order/properties-alphabetical-order": null,
+    "plugin/no-unsupported-browser-features": [true, {
```

Comment:
I also just realized you updated these rules, please update the docs for the rules as well!

Reply:
Also to think about, I believe gulp will exit with a status code of `1` on anything other than success which will cause our travis builds to fail. Do we want this to happen?

Reply:
Are you saying that the way the code is written will cause the build to break, or are you asking as part of the process should this truly cause an error not just a warning and stop the build?

Reply:
I want you make you aware that even though you're changing this severity level to "warning" the travis build will still _error_.

Reply:
Damn. I mean if your browser isnt supported then it _should_ be something that is known & corrected but I'm on the fence of adding something that slows things down... Thoughts?